freshmaker/handlers/koji/rebuild_flatpak_application_on_module_ready.py

```diff
@@ -90,8 +90,8 @@ def handle(self, event):
         self.set_context(db_event)
 
         self.errata = Errata()
-        self.advisory_module_nvrs = self.errata.get_cve_affected_build_nvrs(
```

Comment:
If `get_cve_affected_build_nvrs()` is no longer used, remove also the definition.

Reply:
Ack. Remove get_cve_affected_build_nvrs() and related tests.